integrations/block-data-api.php

```diff
@@ -22,6 +22,14 @@ class BlockDataApiIntegration extends Integration {
 	 */
 	protected string $version = '1.0';
 
+	/**
+	 * Returns `true` if `Block Data API` is already available via customer code. We will use
+	 * this function to prevent activating of integration from platform side.
+	 */
+	public function is_integration_already_available_via_customer(): bool {
+		return defined( 'VIP_BLOCK_DATA_API_LOADED' );
+	}
```

Comment:
Added `is_integration_already_available_via_customer` function in our base Integration class to make sure that it get defined by each integration and apart from that we can reuse this check if its a function.